### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by eyvindbjartiandersen** [newsboat/newsboat#269]
I would like to be able to have data along with tags for each feed so that I can rank the most/least used feeds. And I would also like to have relevant data on which feed is noisiest/quietest etc. I hope you'll see that this would be a very useful feature for all users.

**Comment by Minoru** [comment]
Hi!

> the most/least used feeds

Do you have any specific criteria in mind? Would "percentage of read articles to the total number of articles" work for you?

> which feed is noisiest/quietest

Would "number of articles posted per day, on average" work for you?

Overall, how would you use all of this, @eyvindbjartiandersen? I recall similar stats in Google Reader, and I only looked at them out of curiosity.

An upcoming release already has a "lastupdate" sort criteria. Be sure to try it in two weeks and tell us what you think!

**Comment by eyvindbjartiandersen** [comment]
Yes thank you both your suggestions would work well. I was just looking for a way
to know which feeds I actually read the most and which feeds the most
activity so that I can sort accordingly. I've read about lastupdated and
it's a good feature and maybe it's something I can try.

**Comment by Minoru** [comment]
Okay! To anyone reading this: I'll merge a PR adding two new sort criteria.
